package.json

```diff
@@ -4,9 +4,12 @@
   "private": true,
   "scripts": {
     "build": "tsc",
-    "start": "nodemon -e ts --exec \"(npm run lint -- --force && tsc && node build/server.js) || true\"",
+    "start": "npm run lint && npm run build && npm-run-all --parallel build:watch lint:watch serve:watch",
     "test": "jest",
-    "lint": "tslint -t stylish 'src/**/*.ts'"
+    "lint": "tslint -t stylish 'src/**/*.ts'",
```

Comment:
`tslint` file mask here is different than in `lint:watch`. Please align. We want to cover all `ts` files here, just like the watcher does.